.ci/kitchen-archlts-py3-pytest

```diff
@@ -0,0 +1,16 @@
+@Library('salt@master-1.7') _
+
+runTestSuite(
+    concurrent_builds: 2,
+    distro_name: 'arch',
+    distro_version: 'lts',
+    env: env,
+    golden_images_branch: 'master',
+    jenkins_slave_label: 'kitchen-slave',
+    nox_env_name: 'pytest-zeromq',
+    nox_passthrough_opts: '',
```

Comment:
no passthrough opts?  are we testing everything now instead of just pkg tests?

Reply:
We are. As part of the upcoming 3.8 support.

Reply:
should we add on --ssh-tests then ?

Reply:
Sure. I can't update it today though.
Want to push that change in?

Reply:
sure